sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/Features.cs

```diff
@@ -98,17 +94,9 @@ public async Task<HttpOperationResponse<IList<PatternFeatureInfo>>> ListApplicat
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "versionId");
             }
-            if (skip < 0)
```

Comment:
Why remove the validation?

Reply:
It's not removed, that's another api. The api with skip and take still has their validation